serverlessworkflow/antora.yml

```diff
@@ -1,6 +1,8 @@
 name: serverlessworkflow
 title: "Kogito Serverless Workflow Guides"
 version: main
+display_version: snapshot
+prerelease: snapshot
```

Comment:
where are used those ?

Reply:
`display_version` is what we use to display the version name in the UI, so when you choose the select box in the nav bar, you will see `snapshot` rather than `main`. 

`prerelease` means that this version is not released yet, so Antora won't display it by default when you land on the docs website, but the latest version based on semver.